cache/cache_reservation_manager.h

```diff
@@ -92,8 +111,11 @@ class CacheReservationManager
   //         On keeping dummy entries the same, it always returns Status::OK().
   Status UpdateCacheReservation(std::size_t new_memory_used);
 
-  // One of the two ways of reserving/releasing cache,
-  // see CacheReservationManager::UpdateCacheReservation() for the other.
+  // One of the two ways of reserving cache, used with
+  // ReleaseCacheReservation().
```

Comment:
Does the client need to know about ReleaseCacheReservation()? I thought the client deletes the handle.

Reply:
See above.

Reply:
Done - see above.